python/paddle/fluid/contrib/tests/test_calibration.py

```diff
@@ -199,7 +199,6 @@ def download_model(self):
 
     def run_program(self, model_path, generate_int8=False, algo='direct'):
         image_shape = [3, 224, 224]
-        os.environ['FLAGS_use_mkldnn'] = 'True'
```

Comment:
Since there is `os.environ['FLAGS_use_mkldnn'] = True`, do you need `FLAGS_use_mkldnn=true python test_calibration.py` again?

Reply:
@luotao1 Yes, we need to specify FLAGS_use_mkldnn=true by running python test_calibration.py. The reasons are following:
Calibration tool requires FP32 optimized model by Transpiler. In inference_transpiler.py, it reads the environment (L62): use_mkldnn = bool(os.getenv("FLAGS_use_mkldnn", False)). That's why we can use os.environment in calibration stage, but have to write those lines of code in predication stage (API only available in Pass). So, this PR should be a better and unified solution for Python-based calibration tool.